contacts/storage.go

```diff
@@ -0,0 +1,31 @@
+package contacts
+
+import (
+	"github.com/asdine/storm"
+)
+
+type Contact struct {
```

Comment:
you have to specify the id for storm and the id key should be unique I guess. storm has some docs about there constrains.

Reply:
Done